doc_admin/AdminManual.adoc

```diff
@@ -276,34 +304,34 @@ mysql>FLUSH PRIVILEGES;
 Query OK, 0 rows affected (0.00 sec)
 ----
 
-When we are through, we exit the MySQL client command line with the command:
+NOTE: Remember to set the updated database password also in the openhospital configuration files.
```

Comment:
Should this be `Open Hospital` configuration files` instead of `openhospital configuration files`?  I find no other references to `openhospital configuration`.

Reply:
```suggestion
NOTE: Remember to set the updated database password also in the Open Hospital configuration files.
```